Model/ResourceModel/Menu/Collection.php

```diff
@@ -5,6 +5,8 @@
 
 class Collection extends AbstractCollection
 {
+    protected $_idFieldName = 'menu_id';
```

Comment:
Why is it necessary?

Reply:
It was before when I was testing other solution, now I guess it's not necessary anymore so I deleted it.